3rdparty/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
-if (ECM_FOUND)
-    include(ECMEnableSanitizers)
-endif()
+if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
Please introduce a variable for this, instead of repeating the check multiple times. Something like BUILD_TRACKER